xknx/dpt/dpt_4byte_float.py

```diff
@@ -45,43 +45,331 @@ def to_knx(cls, value):
             raise ConversionError("Cant serialize %s" % cls.__name__, vlaue=value)
 
 
+class DPTAcceleration(DPT4ByteFloat):
+    """DPT 14,000 DPT_Value_Acceleration (ms-2)."""
+
+    unit = "m/s²"
+
+
+class DPTAccelerationAngular(DPT4ByteFloat):
+    """DPT 14,001 DPT_Value_Acceleration_Angular (rad s-2)."""
+
+    unit = "rad/s²"
+
+
+class DPTActivationEnergy(DPT4ByteFloat):
+    """DPT 14,002 DPT_Value_Activation_Energy (J mol-1)."""
+
+    unit = "J/mol"
+
+
+class DPTActivity(DPT4ByteFloat):
+    """DPT 14,003 DPT_Value_Activity (s-1)."""
+
+    unit = "s⁻¹"
+
+
+class DPTMol(DPT4ByteFloat):
+    """DPT 14,004 DPT_Value_Mol (mol)."""
+
+    unit = "mol"
+
+
+class DPTAmplitude(DPT4ByteFloat):
+    """DPT 14,005 DPT_Value_Amplitude """
+
+
+class DPTAngleRad(DPT4ByteFloat):
+    """DPT 14,006 DPT_Value_AngleRad (rad)."""
+
+    unit = "rad"
+
+
+class DPTAngleDeg(DPT4ByteFloat):
+    """DPT 14,007 DPT_Value_AngleDeg ((degree))."""
+
+    unit = "°"
+
+
+class DPTAngularMomentum(DPT4ByteFloat):
+    """DPT 14,008 DPT_Value_Angular_Momentum (J s)."""
+
+    unit = "J s"
+
+
+class DPTAngularVelocity(DPT4ByteFloat):
+    """DPT 14,009 DPT_Value_Angular_Velocity """
```

Comment:
Docstrings should end with a period. https://travis-ci.org/XKNX/xknx/jobs/637620014?utm_medium=notification&utm_source=github_status